src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -52,47 +52,47 @@ const CreateConceptForm = props => (
           <div className="form-group col-md-5">
             <label htmlFor="class">Class</label>
             <div>
-              { (props.concept.toString().trim() === 'symptom-finding') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control symptom-finding"
-                  required
-                  onChange={props.handleChange}
-                >
-                  <option key="Symptom-Finding">Symptom-Finding</option>
-                  <option key="Symptom">Symptom</option>
-                  <option key="Finding">Finding</option>
-
-                </select>
-              </React.Fragment>
+              {(props.concept.toString().trim() === 'symptom-finding') &&
+                <React.Fragment>
+                  <select
+                    id="class"
+                    name="concept_class"
+                    value={props.state.concept_class}
+                    className="form-control symptom-finding"
+                    required
+                    onChange={props.handleChange}
+                  >
+                    <option key="Symptom-Finding">Symptom-Finding</option>
+                    <option key="Symptom">Symptom</option>
+                    <option key="Finding">Finding</option>
+                  </select>
+                </React.Fragment>
               }
-
-              { (props.concept.toString().trim() === 'set') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control set"
```

Comment:
Same here

Reply:
All this are formatting changes that I was reverting. They are not part of this ticket.

Reply:
But i see them in this commit. This pull request should not have any change that is not related to what the ticket requires. If you need formatting, use a new ticket and pull request.

Reply:
I can just create a ticket for just format fixing?

Reply:
yes you can

Reply:
Let me make the changes I will get back to you in a few.